docs/using-pretrained-models.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Using pretrained models
+nav_order: 9
+---
+
+# Using Pretrained Models
+
+Pretrained models are machine learning models trained previously that can be used as the starting point for your specific training tasks.
+
+## Configuration Parameters
+
+To download and use models from previous training runs, or stored elsewhere, use the `pretrained-models` configuration parameter. The keys in this parameter correspond to the training task kinds capable of using pretrained models.
+
+```yaml
+"pretrained-models":
+  {
+    # Configuration for train-teacher kind
+    "train-teacher":
+      {
+        "urls": ["https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/task-id/artifacts/public/build"],
+        "mode": "continue",
+        "type": "default",
+      },
+    # Configuration for train-backwards kind
+    "train-backwards":
+      {
+        "urls": ["https://storage.googleapis.com/bucket-name/models/ru-en/backward"],
+        "mode": "use",
+        "type": "default",
+      },
+  }
+```
+
+### The URL Key
+
+The `urls` key specifies the locations from which the models are downloaded.
+
+### The Mode Key
+
+#### Use Mode
+
+In `use` mode, the pipeline only downloads the model without further training. The tasks that depend on the training task will use the downloaded model artifacts as they are.
+
+#### Continue Mode
+
+In `continue` mode the pipeline uses the downloaded model artifacts from the previous training run as a "checkpoint" and continues training.
```

Comment:
I think it's worth mentioning how continue is different from init. The difference in usage is mostly training the same model on the same corpus versus fine-tuning an existing model on a different corpus.